website/docusaurus.config.js

```diff
@@ -159,6 +159,9 @@ module.exports = {
               label: 'Twitter',
               href: 'https://twitter.com/docusaurus',
             },
+            {
+              html: `<a href="https://www.netlify.com"><img src="https://www.netlify.com/img/global/badges/netlify-dark.svg"/></a>`,
```

Comment:
Maybe use this one? https://www.netlify.com/img/global/badges/netlify-color-accent.svg

Reply:
hmmm. which one is better

before
![image](https://user-images.githubusercontent.com/17883920/71437082-a216dc80-2722-11ea-94ba-dabce36fbd5a.png)

after
![image](https://user-images.githubusercontent.com/17883920/71437061-8dd2df80-2722-11ea-9bbd-2daaabf5789b.png)